sys-config/grate/grate-1.0.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs
+
+DESCRIPTION="A configuration for portage to make the insatll from grate easy"
```

Comment:
s/insatll/install

Reply:
DO you want . version ebuilds to?

Reply:
Not in this PR, maybe later. I'll probably need to issue release versions first.

Reply:
okej . then i can merge it ?

Reply:
> Not in this PR, maybe later. I'll probably need to issue release versions first.

the release versions will be the latest git release you made in a zip and have the version name from Xorg.0.log + plus a date like my grate-sources: https://github.com/hedmo/hedmos-overlay/blob/master/sys-kernel/grate-sources/grate-sources-5.12.0_p20210504.ebuild

Reply:
The PR looks good to me, please merge it. I'll test it all during the next days. Thank you very much!

The git-release is a good option, I have no objections to it. Please feel free to add it!